src/toInteger32.js

```diff
@@ -0,0 +1,21 @@
+import { curryN } from 'ramda';
+/**
+ * Converts double-precision 64-bit binary format IEEE 754 to signed 32 bit integer number.
+ *
+ * @func toInteger32
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/1.12.0|v1.12.0}
+ * @category Function
+ * @sig Number -> Number
+ * @param {Number} number A number
+ * @return {Number} A 32-bit integer number
+ * @see {@link http://speakingjs.com/es5/ch11.html#integers_via_bitwise_operators}
+ * @example
+ *
+ * const n = Math.pow(2, 32) + 1;
+ * const as32BitInteger = toInteger32(n); // => 1
```

Comment:
I'd replace the example with the following one:
```js
RA.toInteger32(2 ** 35); // => 0
RA.toInteger32(2 ** 30); // => 1073741824
```

Reply:
👍